TikTokTrainer/Data/DataProcessing.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  DataProcessing.swift
+//  TikTokTrainer
+//
+//  Created by Ankit  on 2/18/21.
+//
+
+import Foundation
+import SwiftUI
+import AVKit
+import Vision
+
+/// Process the PoseNet overlays on the pre-recorded and user-recorded videos and return a score
+struct ScoringFunction {
+    var preRecordedVid: ProcessedVideo?
+    var recordedVid: ProcessedVideo?
+
+    let jointTriples: [(VNRecognizedPointKey, VNRecognizedPointKey, VNRecognizedPointKey)] = [
+        ((VNRecognizedPointKey(rawValue: "left_shoulder_1_joint"),
+          VNRecognizedPointKey(rawValue: "left_forearm_joint"),
+          VNRecognizedPointKey(rawValue: "left_hand_joint"))),
+        ((VNRecognizedPointKey(rawValue: "right_shoulder_1_joint"),
+          VNRecognizedPointKey(rawValue: "right_forearm_joint"),
+          VNRecognizedPointKey(rawValue: "right_hand_joint"))),
+        ((VNRecognizedPointKey(rawValue: "left_forearm_joint"),
+          VNRecognizedPointKey(rawValue: "left_shoulder_1_joint"),
+          VNRecognizedPointKey(rawValue: "neck_1_joint"))),
+        ((VNRecognizedPointKey(rawValue: "left_shoulder_1_joint"),
+          VNRecognizedPointKey(rawValue: "neck_1_joint"),
+          VNRecognizedPointKey(rawValue: "head_joint"))),
+        ((VNRecognizedPointKey(rawValue: "neck_1_joint"),
+          VNRecognizedPointKey(rawValue: "right_shoulder_1_joint"),
+          VNRecognizedPointKey(rawValue: "right_forearm_joint"))),
+        ((VNRecognizedPointKey(rawValue: "head_joint"),
+          VNRecognizedPointKey(rawValue: "neck_1_joint"),
+          VNRecognizedPointKey(rawValue: "right_shoulder_1_joint")))
+    ]
+    
+    /// Computes angles of PoseNet data with trig
+    /// Cycles through sets of joints to track which angles are available for capture, otherwise angle is marked as 0
+    ///
+    /// - Parameters:
+    ///     - video: The video uploaded by the user and processed by the PoseNetProcessor
+    private func computeAngles(video: ProcessedVideo) -> [[CGFloat]] {
+        var angles = [[CGFloat]]()
+        angles.append([])
+        var anglesIndex = 0
+
+        for slice in video.data {
+            for triple in jointTriples {
+                let pntOne = triple.0
+                let pntTwo = triple.1
+                let pntThree = triple.2
+                var angle: CGFloat = 0
+
+                if slice.points[pntOne] != nil && slice.points[pntTwo] != nil && slice.points[pntThree] != nil {
+                    angle = angleBetweenPoints(leftPoint: slice.points[pntThree]!, middlePoint: slice.points[pntTwo]!, rightPoint: slice.points[pntOne]!)
+                    print(pntTwo.rawValue + ": " + angle.description)
```

Comment:
Can we get rid of most of these print statements? It will clog up the console for debugging stuff later on.